src/components/CustomCollection/CustomBookmark/CustomBookmarkForm.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect } from 'react'
+import { useFormik } from 'formik'
+import * as yup from 'yup'
+import {
+  Form,
+  ButtonGroup,
+  Button,
+  Label,
+  TextInput,
+  Alert,
+  ErrorMessage,
+} from '@trussworks/react-uswds'
+
+type CustomBookmarkFormProps = {
+  onSave: (url: string, label: string) => void
+  onCancel: () => void
+  onDelete?: () => void
+  label?: string
+  url?: string
+  showAddWarning?: boolean
+  nameInputRef: React.RefObject<HTMLInputElement>
+  urlInputRef: React.RefObject<HTMLInputElement>
+}
+export const CustomBookmarkForm = ({
+  onSave,
+  onCancel,
+  onDelete,
+  label,
+  url,
+  showAddWarning,
+  nameInputRef,
+  urlInputRef,
+}: CustomBookmarkFormProps) => {
+  // #TODO: Integrate Formik into our forms following the
+  // wrapper component pattern used in other Truss projects
```

Comment:
question: how have we in the past taken care of TODOs like this? Do we typically create a story/chore? Should we create a follow up story for this? I'm not sure I'm familiar with the wrapper pattern so that can be adding to my confusion.

Reply:
we haven't had a formal process for it, but i think creating a follow-up story is a great idea. i can do that